runtime/src/tasks/fifo/tasks-fifo.c

```diff
@@ -721,7 +717,13 @@ void chpl_task_executeTasksInList(void** p_task_list_void) {
                            child_ptask->id,
                            child_ptask->is_executeOn);
 
-    (*task_to_run_fun)(child_ptask->arg);
+    if (child_ptask->countRunningTasks)
```

Comment:
Don't believe it's correct to increment the running task count here. Conceptually, the parent task (which is what calls this function) is suspended while the children run.  In order to reduce overhead, and with the understanding that all task counting is inherently fuzzy anyway, we thus didn't bother to decrement the count while the parent is waiting, nor to increment it as the parent's thread is used here to execute children.  Is it important for the arg bundle change that the counting adjustment you've coded here be included, or is this just something you thought was missing and so is a "while I was here" change?